schedule/yast/autoyast_disk_as_md_member.yaml

```diff
@@ -29,6 +29,12 @@ test_data:
   mount_point:
     root: '/'
     data: '/data'
-  xpath:
-    mount: '//ns:partitioning/ns:drive[ns:device="/dev/md/0"]/ns:partitions/ns:partition/ns:mount'
-    raid_type: '//ns:partitioning/ns:drive[ns:device="/dev/md/0"]/ns:raid_options/ns:raid_type'
+  cloned_profile:
+    raid_level:
+      xpath: '//ns:partitioning/ns:drive[ns:device="/dev/md/0"]/ns:raid_options/ns:raid_type'
```

Comment:
I've just realized checking existing code in the repo, that I can read this as a list, so I can avoid to have this keys `raid_level`, etc, which would simplify the yaml a lot.

Reply:
done!